doc/bspwm.1

```diff
@@ -2,12 +2,12 @@
 .\"     Title: bspwm
 .\"    Author: [see the "Author" section]
 .\" Generator: DocBook XSL Stylesheets vsnapshot <http://docbook.sf.net/>
-.\"      Date: 04/05/2023
+.\"      Date: 01/01/1980
```

Comment:
That looks fishy :)

Reply:
Same as above. The act of running "make doc" generated that change. I could always revert that line.